Quantum-Control-Applications/Superconducting/Single Fixed Transmon/Use Case 3 - DRAG calibration/README.md

```diff
@@ -0,0 +1,116 @@
+# Derivative Removal by Adiabatic Gate (DRAG) and AC Stark-shift calibration
+
+_Author: Kevin Villegas Rosales_
+
+_Important note: The code in this folder is the exact code that we used
+to run the experiment, so it is tailored for the specific software environment
+and set up. Thus, this is code is for inspirational purposes._
+
+## The goal
+
+Due to the finite anharmonicity of the transmon there is probability to populate the second excited state with
+the application of control pulses.
+
+The goal of this experiment is to calibrate the DRAG coefficient and AC Start shift
+to increase the single qubit gate fidelities as well as to minimize the leakage out of the
+computational space. The single qubit fidelities were quantified with randomized benchmarking (RB).
+
+## The device
+
+The chip contains 5 2D-transoms, where some of them have nearest-neighbor connectivity, 
+with flux tunable lines. For this experiment, we focus only in one transmon with a drive
+to its flux line to put the transmon in the **sweet-spot**. All other transmons are left
+unbiased leaving them effectively at their zero frequency point.
+
+## Method and results
+
+We will show the determination of the DRAG coefficient with two different pulse sequences.
+The first one is related to the application of `(x180)(-x180)` train of pulses and the second
+is a sequence of `x180y90` and `y180x90` pulses. We are calibrating the $\alpha$ parameter in 
+the following formula,
+
+$\Omega^{'}$ = \Omega + i\alpha\frac{d\Omega/dt}{\Delta}$
+
+Also, based on a protocol listed in Ref. [1] we will calibrate the detuning parameters that
+can lead to a minimized leakage out of the computational state. The pulse sequence is a train
+of `(x180)(-x180)`.
+
+Note that we used `cosine` pulses of `16 ns` in length as the envelope for the control. Note that the
+short pulses (16 ns) bandwidth is `~100 MHz`. This requires mixer calibration to reduce the impact
+of LO leakage on the gate fidelity. The IF frequency was `-95 MHz`.
+
+The experiments below were performed with single shot readout, thus we calibrate
+the discrimination between `|g>` and `|e>` state by looking at the IQ blobs. We also
+needed to rotate the IQ blobs to project the information along one axis only for better
+discrimination.
+
+### DRAG coefficient calibration with `x180` `-x180` pulses
```

Comment:
I would put `([[1]](#1))` at the end of the title to clearly separate the two methods